tests/tensor_search/test_backend.py

```diff
@@ -46,6 +46,18 @@ def test_get_index_info_no_index(self):
         except IndexNotFoundError as s:
             pass
 
+    @mock.patch('marqo._httprequests.HttpRequests.get', side_effect=[
```

Comment:
Essentially we mock the `GET _cat/shards/index1` endpoint to return 10 shards, instead of the 5 shards (`NsFields.number_of_shards: 5,`) returned by the default index settings.